src/Functions/GatherUtils/CMakeLists.txt

```diff
@@ -3,6 +3,17 @@ add_headers_and_sources(clickhouse_functions_gatherutils .)
 add_library(clickhouse_functions_gatherutils ${clickhouse_functions_gatherutils_sources} ${clickhouse_functions_gatherutils_headers})
 target_link_libraries(clickhouse_functions_gatherutils PRIVATE dbms)
 
+check_cxx_compiler_flag(suggest-override HAS_SUGGEST_OVERRIDE)
```

Comment:
This is totally wrong.

Reply:
😳

Reply:
Don't worry. It's only reproduced on clang-11 prerelease that I use on my machine :)